### DISCUSSION TITLE
Hotspot icon in default theme

### DISCUSSION THREAD

**Discussion by boris-gu** [OnionUI/Onion#1197] [Q&A] [answered]
I'm starting to make a new theme. I was surprised that the default theme has an icon called "icon-hotspot-wifi.png".  
And this icon is of unusual size 35x35, while other icons are 48x48 in size.

Where can I see the use of this icon? Can the Miyoo Mini+ work as a hotspot?

Also surprised that Blueprint theme, which is advised as a starting point in creating a new theme, does not have this icon

**Comment by XK9274** [comment, marked as answer]
> I'm starting to make a new theme. I was surprised that the default theme has an icon called "icon-hotspot-wifi.png".  
> And this icon is of unusual size 35x35, while other icons are 48x48 in size.
> 
> Where can I see the use of this icon? Can the Miyoo Mini+ work as a hotspot?
> 
> Also surprised that Blueprint theme, which is advised as a starting point in creating a new theme, does not have this icon

Yes the MMP has a hotspot, in onion you can find this in Tweaks -> Networks. The icon doesn't show due to how miyoo handled it but it is possible to make it appear, don't worry about it for now.

**Reply by boris-gu** [reply]
Should I create it now? If so, should it be 48x48 in size?

**Reply by XK9274** [reply]
Might be worthwhile ya. We've asked Miyoo to give us a method for turning on the icon (easier than the one we currently have) but there's no ETA on it.. So I guess it's up to you?
